src/overlays/actors/ovl_En_Syateki_Man/z_en_syateki_man.c

```diff
@@ -1,19 +1,53 @@
 #include "z_en_syateki_man.h"
+#include "vt.h"
+#include "overlays/actors/ovl_En_Syateki_Itm/z_en_syateki_itm.h"
 
 #define FLAGS 0x08000019
 
 #define THIS ((EnSyatekiMan*)thisx)
+#define GALLERY ((EnSyatekiItm*)this->actor.parent)
+
+typedef enum {
+    /* 0 */ RESULT_NONE,
+    /* 1 */ RESULT_WINNER,
+    /* 2 */ RESULT_ALMOST,
+    /* 3 */ RESULT_FAILURE,
+    /* 4 */ RESULT_REFUSE
+} EnSyatekiManGameResult;
+
+typedef enum {
+    /* 0 */ TEXT_CHOICE,
+    /* 1 */ TEXT_START_GAME,
```

Comment:
these need prefixes specifying they are for gallery man

Reply:
I was under the impression local enums like this don't need such prefixes.

Reply:
these enums may be local to this file, but say theres for example a TEXT_ enum from the engine in the future, it can cause a conflict